README.md

```diff
@@ -1,162 +1,74 @@
-Selenium Tests for addons.mozilla.org (amo)
-====================
+Tests for addons.mozilla.org (amo)
+==================================
 
-Thank you for checking out Mozilla's Addon-Tests test suite. Mozilla and the Mozwebqa team are grateful for the help and hard work of many contributors like yourself.
-The following contributors have submitted pull requests to Addon-Tests:
-
-https://github.com/mozilla/Addon-Tests/contributors
+Thank you for checking out Mozilla's Addon-Tests test suite. This repository contains Selenium tests used to test the website addons.mozilla.org.
 
-Continuous Integration
-----------------------
-
-[![Build Status](https://secure.travis-ci.org/mozilla/Addon-Tests.png?branch=master)](http://travis-ci.org/mozilla/Addon-Tests/)
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/Addon-Tests/blob/master/LICENSE)
+[![travis](https://img.shields.io/travis/mozilla/Addon-Tests.svg?label=travis)](http://travis-ci.org/mozilla/Addon-Tests/)
+[![dev](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.dev.svg?label=dev)](https://webqa-ci.mozilla.com/view/AMO/job/amo.dev/)
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.stage.svg?label=stage)](https://webqa-ci.mozilla.com/view/AMO/job/amo.stage/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.prod.svg?label=prod)](https://webqa-ci.mozilla.com/view/AMO/job/amo.prod/)
 
 Getting involved as a contributor
```

Comment:
I think 'as a contributor' is implied, and don't see that it's needed in the heading.